scenario_specifcation.md

```diff
@@ -0,0 +1,129 @@
+## ASSUMPTIONS, PARAMETERS, MODELS, AND REFERENCES:
+
+1. The compositions, core mass and reactor ratio is referenced from [appendix B of E&S study](https://fuelcycleevaluation.inl.gov/Shared%20Documents/ES%20Appendix%20B.pdf).
+
+2. The assumptions of net installed power increase and advanced technology availability is referenced from [FCO study](https://github.com/arfc/transition-scenarios/files/796778/5061-.final.pdf).
+
+3. `CYCAMORE::REACTOR`, the recipe reactor, was used for the reactor archetype.
+
+4. `CYCAMORE::MIXER`, the fixed ratio mixer, was used for the fuel fabrication archetype.
+
+5. EG23 and EG24 are done using an approximate recipe from ORGIEN.
+
+6. EG29 and EG30 are done using a simplification (4 group - U, TRU, MA, FP) model.
+
+7. For SFRs, blanket and driver are averaged into one 'core' (added mass with weighted composition).
+
+## GAPS:
+
+1. Recipe Reactor Limitations:
+   * Can't match with Burnup, needs recipe (always same composition out)
+
+2. Fuel Cycle Facility limitations:
+   * Simple infinite capacity / throughput
+        instead of deployment of multiple facilities depending on need.
+   * FuelFab cannot mix three commodities (Nat U, Rep U, Rep TRU/Pu)
+
+3. FR modeling limitations:
+   * blanket and driver composition has to be averaged
+   * Separate EFPY for blanket and driver can't be modeled
+
+4. Can't Match Demand:
+   * Demand of FR fuel can't be met, and the FR just gets deployed, but does not produce power
+
+5. EG29, 30 reprocessed material only goes in one mixer if both mixer has infinite throughput / buffer.
+
+6. Throughput should be adjusted according to fuel demand, in order for the distribution
+of fissile commodity (rep U, TRU, PU) among different fuel fab facilities. (inf throughput
+ -> one facility gets all fissile commodity)
+
+
+
+## FROM APPENDIX B OF FUELCYCLEEVALUATION.INL.GOV
+### EG23
+Continuous recycle of Pu/U in SFR:
+
+| Category  | Driver| Blanket | 
+| :-------------: |:-------------:| -----:|
+| Material | Pu / Rep U | Nat U |
+| Burnup | 81.5 Gwd/t | 23.5 Gwd/t | 
+| Composition |15.3 % Pu | .7% U235 |  
+| Fuel Residence Time | 3.6 years (avg) |
+| Normalized % |91.21% | 8.79 % |
+
+For 100 GW-year (250 FR-year), uses 1,257,400 kg of fuel  
+-> 1 FR spends 1,257,400/250 * (14/12) = **5867 kg / Batch**  
```

Comment:
This sentence is difficult to follow. 
- For 100GW-year, what uses 1.3 million kg of fuel?
- Perhaps this very large number should be reported in mega-tons  or tons rather than millions of kg. Or, just use scientific notation.